tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -56,44 +58,75 @@ def lambda_handler(event, context):
         AND
             status_id = %d
     """
-    #Gets the dbconnection info.
+    # Gets the dbconnection info.
 
-    db_connect_info = requests_db.get_dbconnect_info()
+    db_connect_info = get_dbconnect_info()
 
     try:
-        rows = database.single_query(sql, database.db_connect_info, (filename, shared_recovery.OrcaStatus.PENDING))
-    except database.DbError as err:
-        LOGGER.error(f"DbError: {str(err)}")
-        raise DatabaseError(str(err))
+        rows = single_query(sql, db_connect_info, (filename, OrcaStatus.PENDING.value))
+    except Exception as ex:
+        LOGGER.error(ex)
```

Comment:
I would improve the logging message here instead of just passing it the exception and stack trace. Possibly something like "Unable to retrieve {filename} metadata.". You will also want to re-raise the error here so that the code stops.